packages/truffle-db/src/workspace/test/utils.ts

```diff
@@ -16,11 +17,16 @@ export const fixturesDirectory = path.join(
   "fixtures"
 );
 
+const tempDir = tmp.dirSync({ unsafeCleanup: true });
+tmp.setGracefulCleanup();
+
 export class WorkspaceClient {
   private workspace: Workspace;
+  private persistedWorkspace: Workspace;
```

Comment:
Maybe this warrants a comment specifying the purpose of the two workspaces

Reply:
Agreed, adding.